people.rkt

```diff
@@ -635,7 +639,8 @@
           @div[class: "row"]{
             @h3[style: "text-align: center;"]{Former Faculty}
               @alumnus-list[
-                @(alumnus "Will Clinger" #:year 2019)
+                @(alumnus "Will Clinger" #:year 2019
+                          #:personal-site "https://en.wikipedia.org/wiki/William_Clinger_(computer_scientist)")
```

Comment:
Wikipedia is definitely not a personal site

I think this is a better personal page: `http://www.cesura17.net/~will/` but it's currently a 404

(`ccs.neu.edu/~will` redirects to the cesura page)

Reply:
Will said his personal page does not exist anymore :(

Reply:
This appears to be the last good snapshot from the web archive: https://web.archive.org/web/20171217194538/http://www.cesura17.net/~will/

Not sure if it's reasonable to link to it.

Reply:
I think Wikipedia is our best bet here: it's better than nothing. 

Also, nothing in the interface marks this link as "a homepage" per se.

Reply:
Oh, didn't see the Web archive link. That might work too.

Reply:
As I understood it, Will intentionally does not have a webpage anymore. So Jan suggested just using the Wikipedia link.

Reply:
Ok. fwiw I think no link is better.